### ISSUE TITLE
Progressive loading of files

### ISSUE THREAD

**Issue by kgsrirama** [qarmin/czkawka#294]
Is it possible to show progressive loading of search results. It will help users in analyzing the duplicates and change filters. Users can also select files in while search is happening. An UI similar to WinDirStat or Baobab will be a good point.

BTW, is it possible to search empty directory and large files while searching for duplicates

**Comment by qarmin** [comment]
I don't think that this would be possible(I mean it is possible but very very difficult or it would slow a lot of searching), because algorithms aren't so straight.

E.g. in empty folder finder, there are collected folders which doesn't have any files inside(but may contains another folder) and just before the end Czkawka remove folders which contains another folders with files.
